src/ivoc/ocmatrix.cpp

```diff
@@ -282,8 +282,6 @@ void OcSparseMatrix::solv(Vect* in, Vect* out, bool use_lu) {
     if (!lu_ || !use_lu || lu_->rows() != m_.rows()) {
         m_.makeCompressed();
         lu_ = std::make_unique<Eigen::SparseLU<decltype(m_)>>(m_);
-        lu_->analyzePattern(m_);
-        lu_->factorize(m_);
```

Comment:
@alkino : just for future clarity: `If we give a matrix to the SparseLU ctor it already analyze and factorize the matrix` : is this behavior expected or documented? i.e. asking because you said it was only obvious after reading implementation. So just want to be sure that in future they won't change this.

Reply:
If one day they decide to stop doing this (which will be non sense) it will crash because there is several internal tests to find this case.